jgthms/bulma
# Merge pull request #1 from jgthms/master

### PLEASE READ THE FOLLOWING INSTRUCTIONS

Thanks for helping out! 😇

* Pull the latest `master` branch
* Run `npm install` to install all Bulma dependencies
* Make sure your Sass code is compliant with the [Bulma Sass styleguide](https://github.com/jgthms/bulma/blob/master/CONTRIBUTING.md#bulma-sass-styleguide)
* If your PR fixes an issue, reference that issue
* If your PR has lots of commits, **rebase** first
* Run `npm run build` and `npm run docs` before submitting your PR

Please remove any unused content (including these instructions) before submitting your issue.

### Pull Request

Fixes #

Changes proposed:


merge upstream
